web/legacy/api/images/get_app_review_images.ts

```diff
@@ -0,0 +1,166 @@
+import { getSdk as getAppReviewImages } from "@/legacy/api/images/graphql/getAppReviewImages.generated";
+import {
+  errorHasuraQuery,
+  errorNotAllowed,
+  errorResponse,
+} from "@/legacy/backend/errors";
+import { getAPIReviewerGraphqlClient } from "@/legacy/backend/graphql";
+import { protectInternalEndpoint } from "@/legacy/backend/utils";
+import { logger } from "@/lib/logger";
+import { GetObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export type ImageGetAppReviewImagesOutput = {
+  logo_img_url?: string;
+  hero_image_url?: string;
+  showcase_img_urls?: string[];
+};
+
+/**
+ * Used when a reviewer is reviewing an app
+ * @param req
+ * @param res
+ */
+export const handleGetAppReviewImages = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+) => {
+  try {
+    if (!process.env.ASSETS_S3_REGION) {
+      throw new Error("AWS Region must be set.");
+    }
+
+    if (!process.env.ASSETS_S3_BUCKET_NAME) {
+      throw new Error("AWS Bucket Name must be set.");
+    }
+
+    if (!protectInternalEndpoint(req, res)) {
+      return;
+    }
+
+    if (req.method !== "GET") {
+      return errorNotAllowed(req.method, res, req);
+    }
+
+    const body = JSON.parse(req.body);
```

Comment:
@andy-t-wang May be it's a good idea to use yup schema and `validateRequestSchema` from `'web/legacy/backend/utils'` to validate schema.

Reply:
But not sure if we really need it for actions

Reply:
Added yup